app/build.gradle

```diff
@@ -18,13 +18,13 @@ androidGitVersion {
 }
 
 android {
-    compileSdkVersion 29
+    compileSdkVersion 30
     buildToolsVersion "29.0.3"
```

Comment:
Please update the build tools to the appropriate version as well and test the app after that.

Reply:
bumped up to 30.0.0